RELEASE/scripts/autoscend/auto_combat.ash

```diff
@@ -957,7 +957,11 @@ string auto_combatHandler(int round, monster enemy, string text)
 		return useSkill($skill[Apprivoisez La Tortue], false);
 	}
 
-	if(!in_zelda() && canUse($skill[Gulp Latte]) && (get_property("_latteRefillsUsed").to_int() == 0) && !get_property("_latteDrinkUsed").to_boolean())
+	if((!in_zelda() || my_class() == $class[Vampyre]) &&	//paths that do not use MP
+	canUse($skill[Gulp Latte]) &&
+	my_mp() * 2 < my_maxmp()		//gulp latte restores 50% of your MP. do not waste it.
+	// && !get_property("_latteDrinkUsed").to_boolean()	//test if currently available. is it redundant with canUse?
```

Comment:
You're correct, this check is redundant. canUse will check if the skill is available which already covers this and more (Latte is equipped, skill is not restricted etc).

Reply:
Thank you for the confirmation.
I had just finished testing it right now too and it was a success. So I removed the commented out part.